tests/test_data/projectgezond.nl/projectgezond.json

```diff
@@ -23,28 +23,33 @@
     "Snijd de plakken ontbijtspek in stukjes en voeg toe.",
-    "Laat het gerecht ongeveer 2 uur stoven met de deksel op de pan. ",
-    "Boen de champignons schoon en snijd ze in kwarten. ",
-    "Snijd de zilveruitjes doormidden. ",
+    "Laat het gerecht ongeveer 2 uur stoven met de deksel op de pan.",
+    "Boen de champignons schoon en snijd ze in kwarten.",
+    "Snijd de zilveruitjes doormidden.",
     "Voeg de champignons en de zilveruitjes toe.",
-    "Laat alles nog minimaal 30 minuten stoven. Doe dit eventueel zonder deksel op de pan, zodat de boeuf bourguignon wat verder inkookt. ",
+    "Laat alles nog minimaal 30 minuten stoven. Doe dit eventueel zonder deksel op de pan, zodat de boeuf bourguignon wat verder inkookt.",
     "Haal het laurierblaadje, het takje tijm en de kruidnagel uit de pan."
   ],
-  "category": "Alle recepten uit ons kookboek 'Altijd lekker', Diner, Kerstrecepten",
-  "yields": "twee personen",
-  "description": "Wist je dat dit recept ook te vinden is in ons bestseller kookboek ‘Altijd lekker’? In dit boek vind je een selectie van 100 populaire recepten uit ons online afslankprogramma. Benieuwd naar alle recepten uit dit boek? Bekijk dan deze pagina.\nDeze klassieker vindt zijn oorsprong in de Franse keuken. ‘Rund op bourgondische wijze’ is een vertaling van ‘Boeuf bourguignon’ die niets aan de verbeelding overlaat.\nDit recept is dan ook het perfecte antwoord als het weer tijd is voor een potje stoof!\nWant het is verre van moeilijk om deze ultieme stoofpot te bereiden. Je hebt enkel wat (wacht)tijd en dus geduld nodig. Het is helemaal geen gek idee dat je dit recept al de dag van tevoren klaarmaakt trouwens. De smaken kunnen dan zelfs nog beter intrekken.\nAls dat geen ‘Boeuf bourguignon’ wordt…",
-  "total_time": "30 minuten + 2 uur stoven",
-  "cuisine": null,
-  "ratings": null,
-  "image": "https://www.projectgezond.nl/wp-content/uploads/2021/11/BoeufBourguignon-768x1024.jpg"
+  "yields": "2 servings",
+  "description": "Wist je dat dit recept ook te vinden is in ons bestseller kookboek ‘Altijd lekker’? In dit boek vind je een selectie van 100 populaire recepten uit ons online afslankprogramma. Benieuwd naar alle recepten uit dit boek? Bekijk dan deze pagina. Deze klassieker vindt zijn oorsprong in de Franse keuken. ‘Rund op bourgondische wijze’ is een vertaling van ‘Boeuf bourguignon’ die niets aan de verbeelding overlaat. Dit recept is dan ook het perfecte antwoord als het weer tijd is voor een potje stoof! Want het is verre van moeilijk om deze ultieme stoofpot te bereiden. Je hebt enkel wat (wacht)tijd en dus geduld nodig. Het is helemaal geen gek idee dat je dit recept al de dag van tevoren klaarmaakt trouwens. De smaken kunnen dan zelfs nog beter intrekken. Als dat geen ‘Boeuf bourguignon’ wordt…",
+  "total_time": 150,
+  "prep_time": 30,
+  "nutrients": {
+    "calories": "428 Kcal",
+    "fatContent": "19 gram Vet",
```

Comment:
This page uses a non-JSON representation of `schema.org` metadata; instead HTML attributes (`itemprop` for example) are used to encode the data.. but something is a bit messy about the data here, because the unit name is redundant information (currently: `fat: 30 g fat` instead of the preferable: `fat: 30 g`).

I'd be willing to reimplement the `nutrients` retrieval here as HTML-based, perhaps after merge if that's easiest process-wise.